src/AppPagination.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { AppFooterSpan } from './components/AppFooterSpan';
+
+interface Pagination {
+    postsPerPage: number;
+    totalPosts: number;
+    paginate: (arg1: number) => void;
+    currentPage: number;
+}
+
+export default function Pagination({ postsPerPage, totalPosts, paginate, currentPage }: Pagination) {
+    const pageNumbers = [];
+    for (let i = 1; i <= Math.ceil(totalPosts / postsPerPage); i++) {
```

Comment:
use .map instead

Reply:
don't know how

Reply:
I think current solution is fine. Paging item,s need to be calculated, also now sure if its possible doing it differently. For me it can stay as it is.

Reply:
left as it is after lukaszblasz confirmation